src/BarChart/BarChart.tsx

```diff
@@ -413,16 +433,16 @@ export const BarChart: FC<Partial<BarChartProps>> = ({
       height={height}
       margins={margins}
       containerClassName={containerClassName}
-      xAxisVisible={isAxisVisible(xAxis.props)}
-      yAxisVisible={isAxisVisible(yAxis.props)}
-      className={classNames(css.barChart, className, css[series.props.type])}
+      xAxisVisible={isAxisVisible(xAxisProps)}
+      yAxisVisible={isAxisVisible(yAxisProps)}
+      className={classNames(css.barChart, className, css[seriesProps.type])}
     >
       {renderChart}
     </ChartContainer>
   );
 };
 
-BarChart.defaultProps = {
+export const BarChartDefaultProps = {
```

Comment:
@steppy452 @amcdnl What name will be correct for that, because I just remove the dot to keep it simple for compare but we need correct naming for that.
I think about these names, maybe some of one have other idea for names?
- barChartDefaultProps
- BAR_CHART_DEFAULT_PROPS

Reply:
It should be `barChartDefaultProps` rather than title cased. https://github.com/svengreb/styleguide-javascript/blob/main/rules/naming_conventions.md#camelcase